### ISSUE TITLE
Using a target size ... that is different to the input size

### ISSUE THREAD

**Issue by zolekode** [deepset-ai/FARM#229]
Please what does this mean ?

Train epoch 1/20 (Cur. train loss: 218.7825):  61%|██████▏   | 62/101 [01:40<02:25,  3.72s/it]/home/ec2-user/anaconda3/envs/python3/lib/python3.6/site-packages/torch/nn/modules/loss.py:431: UserWarning: **Using a target size (torch.Size([32, 1])) that is different to the input size (torch.Size([32]))**. This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.

**Comment by tholor** [comment]
I suppose the shape of your predictions are not matching the shape of your labels.Can you please provide some additional context: What task are you working on? How do you initialize prediction head and processor? How can we reproduce this? 
Thanks :)

**Comment by zolekode** [comment]
I am working on a regression task (The labels range from 0 to 16). Here is how i initialize the prediction head:
```prediction_head = RegressionHead([768, 1])```
Basically, the code is very similar to that of the doc_regression.py found under examples. Nothing significant has changed (data, max_sequence_length, lr and model='distilBERT)

**Comment by tholor** [comment]
Ok, found it: the logits were squeezed and therefore didn't match the current implementation of labels.
Fixed it in #230. Can you please verify that your warning message is gone with this fix?
